src/utilities/wishlistFun.js

```diff
@@ -0,0 +1,15 @@
+export const wishlistFun = (wishlistState, action) => {
+    switch(action.type){
+        case ("ADD-TO-WISHLIST"):
+            if(wishlistState.items.filter(item => item._id === action.payload._id).length !== 0){ 
```

Comment:
Take care of formatting your code, use a code formatter

Reply:
done ✅